daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-features-concepts.md

```diff
@@ -126,20 +126,69 @@ To take advantage of the workflow replay technique, your workflow code needs to
 1. **Workflow functions must call deterministic APIs.**  
     APIs that generate random numbers, random UUIDs, or the current date are _non-deterministic_. To work around this limitation, you can:
     - Use these APIs in activity functions, or 
-    - (Preferred) Use built-in equivalent APIs offered by the SDK. For example, each authoring SDK provides an API for retrieving the current time in a deterministic manner.
+    - (Preferred) Use built-in equivalent APIs offered by the SDK. For example, each authoring SDK provides an API for retrieving the current time in a deterministic manner.  
+
+    For example, instead of this:
+
+    ```csharp
```

Comment:
Should we set up language tabs here since we'll want to add examples for multiple languages? Or would you prefer that we add language tabs later, when we actually have multiple languages?

Reply:
good idea - added a tab to prep for languages being added!